### DISCUSSION TITLE
Future support / maintenance plan?

### DISCUSSION THREAD

**Discussion by toqduj** [kip-hart/MicroStructPy#74] [General]
Hi, sorry for the odd question, but what are the future plans for this package? i.e. will it receive continuous TLC, or is/was it linked to a Ph.D. project or a fixed-term post-doc study?

**Comment by kip-hart** [comment]
Hi @toqduj, I'm continuing to maintain the repo. It was linked to my PhD and I probably won't add many more features to it. I'll make sure it still builds and runs as new Python versions are released. All that said, I'm happy to merge in PRs from other people if they write features/improvements.

**Comment by toqduj** [comment]
Thanks for the feedback, I will try to get it working to create a 3D polycrystalline particle (should be well within its capabilities). I'll then try to fill the individual phases with a tiled electron density lattice... Fun stuff 😄

**Reply by kip-hart** [reply]
Wow - that's a very detailed microstructure! Check out the examples page to see if there's something there that can help you get started. If you have any questions please let me know, I'm happy to help.

https://docs.microstructpy.org/en/latest/examples/index.html
